recipes/newrelic/infrastructure/ohi/couchbase/debian.yml

```diff
@@ -45,7 +45,7 @@ inputVars:
     prompt: "Couchbase Query Port (default: 8093)"
     default: 8093
   - name: "NR_CLI_API_USE_SSL"
-    prompt: "Use SSL to connect? (true/false)"
+    prompt: "Use SSL to connect? (false/true)"
```

Comment:
Why rearrange this? Is it due to how default is `false`? Although minor, this might throw some users off (me being one of them)

Reply:
Correct, I re-ordered to match our default (false), so the first value is the default. It seemed backward otherwise.

Reply:
I agree, but also, its a small issue. What if we did:

`prompt: "Use SSL to connect? (default: false)"`
or
`prompt: "Use SSL to connect? (true/false, default: false)`

Reply:
I don't really favor any of those 2 alternatives. I think the user should know what are all the possible choices (discard first suggestion).
The 2nd one is quite verbose, we know users don't like to read text. My concern is they stop to read at the first param and use that one, which has plenty of other implications and would put them on a path to fail because they don't have all the required certificate and pem key files (3 files in total).